src/views/workflow-history/__tests__/workflow-history.test.tsx

```diff
@@ -67,32 +68,32 @@ describe('WorkflowHistory', () => {
 
   it('throws an error if the request fails', async () => {
     try {
-      await act(() => setup({ error: true }));
+      act(() => setup({ error: true }));
```

Comment:
There is no need for act without await.
What are you trying to achieve here ?

Reply:
You're right, I'll add it back.
The initial act() wrapping the userEvent had this error, so I mistakenly assumed act() always returned void.
<img width="569" alt="Screenshot 2024-10-25 at 2 46 55 PM" src="https://github.com/user-attachments/assets/431c34f7-2332-491f-bf9a-c34806d449bd">